src/superphot_plus/ztf_transient_fit.py

```diff
@@ -144,23 +154,16 @@ def create_prior(cube):
         # Compute the truncated Gaussian distribution for all values at once.
         tg_vals = truncnorm.ppf(cube, tg_a, tg_b, loc=prior_mean, scale=prior_std)
 
-        cube[0] = max_flux * 10 ** tg_vals[0]  # log-uniform for A from 1.0x to 16x of max flux
-        cube[1] = tg_vals[1]  # beta UPDATED, looks more Lorentzian so widened by 1.5x
-        cube[2] = 10 ** tg_vals[2]  # very broad Gaussian temporary solution for gamma
-        cube[3] = tg_vals[3]
-        cube[4] = 10 ** tg_vals[4]  # taurise, UPDATED
-        cube[5] = 10 ** tg_vals[5]  # tau fall UPDATED
-        cube[6] = 10 ** tg_vals[6]  # lognormal for extrasigma, UPDATED
-
-        # g band
-        cube[7] = tg_vals[7]  # A UPDATED
-        cube[8] = tg_vals[8]  # beta UPDATED
-        cube[9] = tg_vals[9]  # gamma, GAUSSIAN not Lorentzian
-        cube[10] = tg_vals[10]  # t0 UPDATED
-        cube[11] = tg_vals[11]  # taurise UPDATED, Gaussian
-        cube[12] = tg_vals[12]  # taufall UPDATED
-        cube[13] = tg_vals[13]  # extra sigma UPDATED, Gaussian
+        cube[start_idx] = max_flux * 10 ** tg_vals[start_idx]  # log-uniform for A from 1.0x to 16x of max flux
+        cube[start_idx+1] = tg_vals[start_idx+1]  # beta UPDATED, looks more Lorentzian so widened by 1.5x
+        cube[start_idx+2] = 10 ** tg_vals[start_idx+2]  # very broad Gaussian temporary solution for gamma
+        cube[start_idx+3] = tg_vals[start_idx+3]
+        cube[start_idx+4:start_idx+7] = 10 ** tg_vals[start_idx+4:start_idx+7]  # taurise, UPDATED
```

Comment:
Nice formulation.

You should update the comment to capture that it includes taufall and extrasigma as well.